cmd/app.go

```diff
@@ -561,8 +565,9 @@ func templateInstall(ctx *cli.Context) error {
 			return err
 		}
 
+		interactive := !ctx.Bool("no-prompt")
 		answers := make(map[string]string)
-		err = processAnswers(ctx, c, templateVersion, answers, true)
+		err = processAnswers(ctx, c, templateVersion, answers, interactive)
```

Comment:
if no-prompt is set to false by default, the old behavior of passing true to processAnswers by default won't be preserved.

Reply:
There's a negative above, interactive == !no-prompt

Reply:
ah, my bad